### PR TITLE
Share docs between compound component of the same parent

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
In order to share docs between compound components of the same parent I copy the docs entry from the parent to children 
for example components like (TabController and its children TabController.TabBar, TabController.TabPage, etc...) 

## Changelog
Share docs between compound components

### CODE REVIEW

File: docuilib/src/components/ComponentPage.tsx [hunk 1]
```diff
@@ -79,6 +79,7 @@ export default function ComponentPage({component}) {
         // title: name,
         layout: 'horizontal',
         ...hero,
+        title: component.name,
```
Comment (Inbal-Tish) [inline docuilib/src/components/ComponentPage.tsx:82]: Look at the commented out code. We don't to add the 'incubator' prefix to the incubator components?

**Comment by Inbal-Tish** [comment]
@ethanshar I'm not sure I understand this PR... How will the TabController docs will look like if we want to add docs to both TabBar (exists) and TabPage components?

File: docuilib/src/components/ComponentPage.tsx (hunk 1 above)
Comment (ethanshar) [reply docuilib/src/components/ComponentPage.tsx:82]: I don't see why not
We want developers to know how to access Incubator components, especially if we still have incubator and non-incubator version (like Dialog and Slider)

**Comment by ethanshar** [comment]
> @ethanshar I'm not sure I understand this PR... How will the TabController docs will look like if we want to add docs to both TabBar (exists) and TabPage components?

I discussed with Yulia, she also preferred to remove 🙄 so I removed

File: scripts/docs/buildDocsCommon.js [hunk 2]
```diff
@@ -60,6 +69,11 @@ function processComponents(components) {
     const isParentComponent = parentComponents.includes(componentName);
     const isIncubatorComponent = component.category === 'incubator';
 
+    if (isCompoundComponent(component.name)) {
+      const parentComponent = getParentComponent(component.name, components);
+      component.docs = parentComponent.docs;
```
[the remark this answers is not in the document]
Comment (ethanshar) [reply scripts/docs/buildDocsCommon.js]: I removed this logic

File: docuilib/src/components/ComponentPage.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply docuilib/src/components/ComponentPage.tsx:81]: So use the commented 'name' in the title filed, no?

**Comment by Inbal-Tish** [comment]
> > @ethanshar I'm not sure I understand this PR... How will the TabController docs will look like if we want to add docs to both TabBar (exists) and TabPage components?
> 
> I discussed with Yulia, she also preferred to remove 🙄 so I removed

Then why you did remove the docs for the TabBar after moving them to the TabController? Now there is a duplication...

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**